scripts/ci/install_closure_deps.sh

```diff
@@ -19,13 +20,16 @@ mv $CLANG_BUILD clang
 rm -f $CLANG_TAR
 
 # Install closure compiler and linter.
-if [ ! -d "closure-compiler" ]; then
-  git clone --depth 1 https://github.com/google/closure-compiler.git
+if [ ! -e "closure-compiler-1.0-SNAPSHOT.jar" ]; then
```

Comment:
I think Travis effectively runs on a fresh machine each time, so I would expect this to always return true. If I'm wrong you just might compile using an older compiler, which should generally be fine I suppose.